From: Luca Boccassi <bluca@debian.org>
Date: Mon, 24 Jul 2023 18:06:30 +0100
Subject: ukify: fix armv7 architecture mapping

$ uname -m
armv7l

Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/src/kernel-install/60-ukify.install", line 223, in <module>
    main()
  File "/<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/src/kernel-install/60-ukify.install", line 219, in main
    call_ukify(opts)
  File "/<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/src/kernel-install/60-ukify.install", line 205, in call_ukify
    ukify['finalize_options'](opts2)
  File "/<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/ukify", line 1423, in finalize_options
    opts.efi_arch = guess_efi_arch()
                    ^^^^^^^^^^^^^^^^
  File "/<<PKGBUILDDIR>>/obj-arm-linux-gnueabihf/ukify", line 78, in guess_efi_arch
    raise ValueError(f'Unsupported architecture {arch}')
ValueError: Unsupported architecture armv7l
---
 src/ukify/ukify.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ukify/ukify.py b/src/ukify/ukify.py
index 282f46f..3b4ee51 100755
--- a/src/ukify/ukify.py
+++ b/src/ukify/ukify.py
@@ -59,7 +59,7 @@ EFI_ARCH_MAP = {
     'x86_64'       : ['x64', 'ia32'],
     'i[3456]86'    : ['ia32'],
     'aarch64'      : ['aa64'],
-    'arm[45678]*l' : ['arm'],
+    'armv[45678]*l': ['arm'],
     'loongarch32'  : ['loongarch32'],
     'loongarch64'  : ['loongarch64'],
     'riscv32'      : ['riscv32'],
